README.md

```diff
@@ -1 +1,2 @@
 # jv-homework-template
+Реализовать свой ArrayList исходя из интерфейса https://pastebin.com/wMsifZSt
```

Comment:
@ymilenin @Sofasmile please remove link. Check the jv-linked-list README implementation